<launch>
	<node pkg="dji_sdk" type="dji_sdk_node" name="dji_sdk" output="screen">
		<!-- node parameters -->
		<param name="serial_name" type="string" value="/dev/ttyTHS1"/>
		<param name="baud_rate" type="int" value="230400"/>
		<param name="app_id" type="int" value="1034699"/>
		<param name="app_version" type="int" value="1"/>
		<param name="app_bundle_id" type="string" value="Not required in this SDK version"/>
		<param name="enc_key" type="string" value="40f400d2704802ce6e3267681b1b67bbf5b5d5a188ef9210bfd8e9f197355e2b"/>
		<param name="groundstation_enable" type="int" value="1"/>
	</node>
</launch>
